src/BlazorWebView/src/Maui/Windows/WinUIWebViewManager.cs

```diff
@@ -22,6 +22,19 @@ public class WinUIWebViewManager : WebView2WebViewManager
 		private readonly WebView2Control _webview;
 		private readonly string _hostPageRelativePath;
 		private readonly string _contentRootDir;
+		private static readonly bool _isPackagedApp;
+
+		static WinUIWebViewManager()
```

Comment:
I think there is a helper in essentials... PlatformUtils I think. Might be worth using that just in case they change the rules and we update that and miss this.

Reply:
It's internal, I can't consume it from here.